website/docs/cli/cloud/migrating.html.md

```diff
@@ -0,0 +1,119 @@
+  label to `cloud`. The inner arguments are unchanged.
+
+    ```diff
+     terraform {
+    -  backend "remote" {
+    +  cloud {
+         organization = "my-org"
+
+         workspaces {
+           name = "my-app-prod"
+         }
+       }
+     }
+    ```
+
+- If you were using multiple workspaces via the `prefix` argument, replace it
+  with a `cloud` block that uses the `tags` argument.
+
+    The existing workspaces don't need to already have these tags — when you
+    initialize, Terraform will attempt to add the specified tags to them.
+
+    ```diff
+     terraform {
+    -  backend "remote" {
+    +  cloud {
+         organization = "my-org"
+
+         workspaces {
+    -      prefix = "my-app-"
+    +      tags = ["app:mine"]
+         }
+       }
+     }
+    ```
+
+### Limitations
+
+To take advantage of automatic tagging of existing workspaces, you must have
+permission to change tags for every affected workspace. Thus, switching to the
+`cloud` integration works best when someone with permission to manage workspaces
+for your organization performs the migration first. When other users
+re-initialize later, the workspaces will already be tagged appropriately and
+tagging permissions won't be required.
+
+### Workspace Names
+
+After switching to the `cloud` integration, your local workspace names will
+change to match their remote names.
+
+The `remote` backend was designed to use two different names for each workspace:
+a short name for local use, and a longer name (with a prefix added) for remote
+use. For example, a remote workspace named `app-prod` would be called `prod`
+locally if the backend was configured with a prefix of `app-`.
+
+This proved confusing in practice, so the `cloud` integration doesn't do that.
+Instead, workspaces use the same names remotely and locally.
+
+## Migrating from Local State or Other Backends
+
+If you _weren't_ using the `remote` backend and the working directory already
+has state data available (using either local state or a
+[state backend](/docs/language/settings/backends/index.html)), Terraform will
+ask your approval to migrate that state to Terraform Cloud. This process is
+interactive and self-documenting, and resembles moving between state backends.
+
+There's one unique step in migrating to Terraform Cloud: if you configured your
+`cloud` block using the `tags` strategy, you might need to rename your
+workspaces.
+
+This happens because Terraform Cloud has a global view of your infrastructure
+(and expects workspaces to have unique names), but Terraform CLI only knows
+about one working directory at a time (and doesn't mind if you re-use the same
+workspace names in different directories). If you try to migrate a workspace
+whose name is already claimed for a different purpose, bad things can happen.
+
+To help you avoid this, Terraform will offer you a chance to add a prefix and/or
+suffix to your workspace names when migrating them. You'll need to provide a
```

Comment:
Technically not true, as it can be an infix too!

Reply:
More helpfully: "Terraform will allow you to change your workspace names relative to their existing names via a pattern you specify. The pattern must contain an asterisk (`*`)..."